python/src/FieldFunctionalChaosSobolIndices_doc.i.in

```diff
@@ -0,0 +1,153 @@
+%feature("docstring") OT::FieldFunctionalChaosSobolIndices
+"Sobol indices from a functional decomposition.
+
+This class allows to perform sensitivity analysis from field chaos decomposition.
+The process decomposition is done by Karhunen-Loeve and the modes interpolation
+is performed by functional chaos:
+
+.. math::
+
+    \tilde{g}(x) = \sum_{k \in K_c} \beta_{\vect{\alpha}_k} \Psi_{\vect{\alpha}_k}(x)
```

Comment:
There are many undefined notations here and below. Does this come from some other doc page?

Reply:
yes it comes form the FieldFunctionalChaosAlgorithm mostly

Reply:
> There are many undefined notations here and below. Does this come from some other doc page?

> > yes it comes form the FieldFunctionalChaosAlgorithm mostly

Could you add a sentence to tell users where they can find the definitions of the notations (with a link to these pages)?